### PR TITLE
Improve parsing of performance test logs

### PR DESCRIPTION
**Pull request by goodlyrottenapple**
This should allow us to store only the `.log` files and do the ` call    ` filtering inside the python script, which should help de-clutter the produced files as the number of log files grows quickly....

### CODE REVIEW

File: scripts/performance-tests-kevm.sh [hunk 1]
```diff
@@ -54,12 +55,10 @@ feature_shell "make poetry && poetry run -C kevm-pyk -- kevm-dist --verbose buil
 feature_shell "make test-prove-pyk PYTEST_PARALLEL=$PYTEST_PARALLEL PYTEST_ARGS='--maxfail=0 --timeout 7200 -vv --use-booster' | tee $SCRIPT_DIR/kevm-$KEVM_VERSION-$FEATURE_BRANCH_NAME.log"
 killall kore-rpc-booster || echo "No zombie processes found"
 
-if [ ! -e "$SCRIPT_DIR/kevm-$KEVM_VERSION-master-$MASTER_COMMIT.log" ]; then
-  master_shell "make test-prove-pyk PYTEST_PARALLEL=$PYTEST_PARALLEL PYTEST_ARGS='--maxfail=0 --timeout 7200 -vv --use-booster' | tee $SCRIPT_DIR/kevm-$KEVM_VERSION-master-$MASTER_COMMIT.log"
+if [ ! -e "$SCRIPT_DIR/kevm-$KEVM_VERSION-master-$MASTER_COMMIT_SHORT.log" ]; then
+  master_shell "make test-prove-pyk PYTEST_PARALLEL=$PYTEST_PARALLEL PYTEST_ARGS='--maxfail=0 --timeout 7200 -vv --use-booster' | tee $SCRIPT_DIR/kevm-$KEVM_VERSION-master-$MASTER_COMMIT_SHORT.log"
```
Comment (jberthold) [inline scripts/performance-tests-kevm.sh:59]: While we are at it, could we make a dedicated (`.gitignore`-d) directory for the log data?
We would replace `SCRIPT_DIR` here by `LOG_DIR` (defaults to `realpath ${SCRIPT_DIR}/../log-store` or similar), and `cd` into that dir below, calling `python3 ${SCRIPT_DIR}/compare.py`; likewise in the other script

File: scripts/performance-tests-kontrol.sh [hunk 2]
```diff
@@ -10,6 +10,7 @@ KONTROL_VERSION=${KONTROL_VERSION:-'v0.1.49'}
 SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
```
Comment (jberthold) [inline scripts/performance-tests-kontrol.sh:10]: Also while we are here, `KONTROL_VERSION` is still hard-wired to 0.1.49 in this script and should be switched to `master` as in the `kevm` script (also changing how we build things).

**Review by jberthold — COMMENTED** [review]
:+1: What is here LGTM. Suggested a few more tweaks

**Review by geo2a — APPROVED** [review]
I agree with Jost's suggestions, pre-approving!

File: scripts/performance-tests-kontrol.sh [hunk 3]
```diff
@@ -62,15 +70,15 @@ master_shell() {
 
 feature_shell "poetry install && poetry run kevm-dist --verbose build plugin haskell foundry --jobs 4"
```
Comment (jberthold) [inline scripts/performance-tests-kontrol.sh:71]: Should the build targets get prefixed here, as in the `kevm` script?
```suggestion
feature_shell "poetry install && poetry run kevm-dist --verbose build evm-semantics.plugin evm-semantics.haskell foundry --jobs 4"
```

Comment (goodlyrottenapple) [reply scripts/performance-tests-kontrol.sh:71]: ah probably. im just testing them before merging so will correct

**Pull request merged**